### ISSUE TITLE
Cannot run scancode CLI

### ISSUE THREAD

**Issue by nishakm** [nexB/scancode-toolkit#2529]
### Description
@dlorenc and I  tried to run scancode on a go module but get this error:
```
root@c87f9ac76eb9:/# scancode
Traceback (most recent call last):
  File "/usr/local/bin/scancode", line 5, in <module>
    from scancode.cli import scancode
  File "/usr/local/lib/python3.9/site-packages/scancode/cli.py", line 108, in <module>
    plugin_classes, plugin_options = PluginManager.load_plugins()
  File "/usr/local/lib/python3.9/site-packages/plugincode/__init__.py", line 207, in load_plugins
    mgr_setup = manager.setup()
  File "/usr/local/lib/python3.9/site-packages/plugincode/__init__.py", line 231, in setup
    self.manager.load_setuptools_entrypoints(entrypoint)
  File "/usr/local/lib/python3.9/site-packages/pluggy/manager.py", line 299, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/usr/local/lib/python3.9/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/usr/local/lib/python3.9/site-packages/summarycode/classify.py", line 58, in <module>
    class FileClassifier(PreScanPlugin):
  File "/usr/local/lib/python3.9/site-packages/summarycode/classify.py", line 103, in FileClassifier
    PluggableCommandLineOption(('--classify',),
  File "/usr/local/lib/python3.9/site-packages/commoncode/cliutils.py", line 373, in __init__
    super(PluggableCommandLineOption, self).__init__(
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 2482, in __init__
    super().__init__(param_decls, type=type, multiple=multiple, **attrs)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 2041, in __init__
    self.type = types.convert_type(type, default)
  File "/usr/local/lib/python3.9/site-packages/click/types.py", line 1019, in convert_type
    return FuncParamType(ty)
  File "/usr/local/lib/python3.9/site-packages/click/types.py", line 158, in __init__
    self.name = func.__name__
AttributeError: 'str' object has no attribute '__name__'
```
Help?

### How To Reproduce

Install scancode-toolkit pip package. Then run `scancode`.

### System configuration

Docker container: https://gist.github.com/dlorenc/905d30bbae11dec25910ea241d21fe63#gistcomment-3753073

**Comment by sebathomas** [comment]
I think it's a conflict with the new click version. I had the same issue after installing via pip in a new virtualenv, and I was able to run scancode by downgrading click to version 7.1.2.

**Comment by pombredanne** [comment]
@sebathomas @nishakm thanks for the report. I am working on a fix!

**Comment by pombredanne** [comment]
This has been fixed in v21.6.7.
Thank for the report.
